### PR TITLE
add license and pkg authors

### LINKED ISSUE #8: Code in TLG-C recipes needs clear licensing
Code is now public facing but no license is stated for it's use, either at the initial splash page or within the individual qmd files, that I can see. 

Also, the DESCRIPTION file in this repo lists the license as `Roche`. 

@pawelru this probably falls to you i'd imagine. cc @cicdguy

### PR DESCRIPTION
**Pull request by pawelru**
close #8

### CODE REVIEW

**Review by shajoezhu — APPROVED** [review]
lgtm! thanks @pawelru

File: index.qmd [hunk 1]
```diff
@@ -11,3 +11,7 @@ date: "`r Sys.Date()`"
 A catalog of Tables, Listings, and Graphs output produced by NEST tools.
 
 **See the full index of available TLG templates on the [Index page](tlg-index.qmd).**
+
+## License
+
+This book as well as code examples are licensed under the Apache License, Version 2.0.
```
Comment (Melkiades) [inline index.qmd:17]: book?

Comment (pawelru) [reply index.qmd:17]: Do you think that "website" is more suitable?

Comment (Melkiades) [reply index.qmd:17]: catalog or repository maybe?

Comment (Melkiades) [reply index.qmd:17]: catalog sounds better

Comment (shajoezhu) [reply index.qmd:17]: catalog i think is good.

Comment (pawelru) [reply index.qmd:17]: ok - will open PR

**Pull request merged**